core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2565,6 +2606,11 @@ fn create_webview<T: UserEvent, F: Fn(RawWindow) + Send + 'static>(
     ..
   } = pending;
 
+  let _webview_create_span = tracing::debug_span!("wry::webview::create").entered();
+  let window_draw_span = tracing::debug_span!("wry::window::draw").entered();
+  let window_create_span =
+    tracing::debug_span!(parent: &window_draw_span, "wry::window::create").entered();
```

Comment:
Do you think these might be better suited to live in the `wry` repo now that we have forked it too?  (in which case we wouldn't need the `wry::` prefix that we can derive from the span metadata)

Reply:
I don't really want to bloat wry with a bunch of tracing calls, but whatever we all decide works for me. cc @Wu-Yu-Wei

Reply:
I think adding tracing to wry makes sense too (and we'll get the request at some point anyway) as even users that use wry just as a library would want to get perf info